package.json

```diff
@@ -33,7 +33,6 @@
     "test:coverage:html": "cross-env NODE_ENV=test BABEL_ENV=coverage nyc mocha 'packages/**/*.test.{js,ts,tsx}' --exclude '**/node_modules/**' && nyc report --reporter=html",
     "test:karma": "cross-env NODE_ENV=test karma start test/karma.conf.js",
     "test:unit": "cross-env NODE_ENV=test node --expose_gc ./node_modules/.bin/mocha 'packages/**/*.test.{js,ts,tsx}' --exclude '**/node_modules/**' --timeout 3000",
-    "test:watch": "yarn test:unit --watch",
```

Comment:
The core does not have that one and I don't think it's worth keeping
A simple `yarn test:unit --watch` works fine.

Reply:
I suspect that the main monorepo used to have `yarn test:watch` but it was replaced with `yarn t xxx`. `yarn t xxx` is superior to `yarn test:unit --watch` when you are working on a single test file, and the codebase has a lot of unit tests that all needs to be loaded before a single test can run, even with `only`.